torch/quantization/fx/quantization_patterns.py

```diff
@@ -1056,3 +1056,34 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
         setattr(quantizer.modules[parent_name], name, quantized_standalone_module)
         quantizer.modules[node.target] = quantized_standalone_module
         return quantizer.quantized_graph.node_copy(node, load_arg(quantized=input_quantized_idxs))
+
+@register_quant_pattern(torch.nn.GELU)
+@register_quant_pattern(torch.nn.functional.gelu)
+class GELUModuleQuantizeHandler(QuantizeHandler):
+    def __init__(self, quantizer: QuantizerCls, node: Node):
+        super().__init__(quantizer, node)
+        self.gelu_node = node
+        if node.op == "call_module":
+            self.gelu = quantizer.modules[self.gelu_node.target]  # next node?
+        elif node.op == "call_function":
+            self.gelu = node.target
+
+    def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
+                is_reference: bool = False,
+                convert_custom_config_dict: Dict[str, Any] = None) -> Node:
+        if not is_reference:
+            warnings.warn("Only reference implementation support at this time")
+        args = load_arg(quantized=[0, 1])(self.gelu_node.args)
```

Comment:
gelu only has one argument I think? https://pytorch.org/docs/stable/nn.functional.html#torch.nn.functional.gelu

you can do load_arg(quantized=[0])